RELEASE-NOTES.txt

```diff
@@ -8,6 +8,7 @@
 -----
 * [**] Added support to use third-party passkey providers and other devices passkeys as a WordPress.com login option [https://github.com/wordpress-mobile/WordPress-Android/pull/20174]
 * [*] [Jetpack-only] Fix the visibility issue with the menu button on the stats [https://github.com/wordpress-mobile/WordPress-Android/pull/20175]
+* [**] Block editor: Introduce VideoPress v5 support, to fix issues using video block with dotcom and Jetpack sites [https://github.com/wordpress-mobile/gutenberg-mobile/pull/6634]
```

Comment:
Version 24.3 is in beta so this release note should be under the 24.4 section.

Reply:
Additionally, since VideoPress v5 is a Jetpack block and it won't be available in the WP app, we should consider adding the prefix `[Jetpack-only]`.

Reply:
Fixed in https://github.com/wordpress-mobile/WordPress-Android/pull/20181/commits/1f5d8b7b5faafb04e6869e21b4658ee32ed9bec0, thank you!